app/src/main/java/com/nutomic/syncthingandroid/activities/FirstStartActivity.java

```diff
@@ -240,7 +240,7 @@ private boolean shouldSkipSlide(Slide slide) {
             case API_LEVEL_30:
                 // Skip if running as root, as that circumvents any Android FS restrictions.
                 return upgradedToApiLevel30()
-                        || PreferenceManager.getDefaultSharedPreferences(this).getBoolean(Constants.PREF_USE_ROOT, false);
```

Comment:
Shouldn't there now be an unused import for `PreferenceManager`? Same on all other files.